doc/nvim-tree-lua.txt

```diff
@@ -114,7 +114,11 @@ function.
       filters = {
 	dotfiles = false,
 	custom = {}
-      }
+      },
+      trash = {
+	cmd = "trash",
+	require_confirm = true,
+      },
```

Comment:
it's badly indented and missing description below just like other options, please finish this properly :)

Reply:
my bad, I was wondering about where to place the description, the bad indents were due to my indentation setup not matching yours.

Reply:
Hello. I just accidentally deleted a file, and unfortunately trash was not installed as well. Did nvim-tree just default to `rm`. Because I can't seem to find it in my `~/.local/share/trash` dir

Reply:
The above is outdated.

See `:help nvim-tree-opts-trash` for the default commands and `:help nvim-tree-opts-ui` for confirmation options.

If the trash fails, you will be notified of the error.